test/e2e/performanceprofile/functests/2_performance_update/test_suite_performance_update_test.go

```diff
@@ -42,10 +41,11 @@ var _ = AfterSuite(func() {
 func TestPerformanceUpdate(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	rr := []Reporter{}
-	if ginkgo_reporters.Polarion.Run {
-		rr = append(rr, &ginkgo_reporters.Polarion)
-	}
-	rr = append(rr, junit.NewJUnitReporter("performance_update"))
-	RunSpecsWithDefaultAndCustomReporters(t, "Performance Addon Operator Update e2e tests", rr)
+	RunSpecs(t, "Performance Addon Operator configuration")
 }
+
+var _ = ReportAfterSuite("e2e serial suite", func(r Report) {
```

Comment:
what is this addition ?

Reply:
its the new way to add reports in ginkgo v2